tests/testthat/test-DataLongitudinal.R

```diff
@@ -27,3 +25,51 @@ test_that("DataLongitudinal being rendered to list is as expected for simple inp
     expect_identical(li$n_lm_time_grid, 201L)
     expect_length(li$lm_time_grid, 201L)
 })
+
+
+test_that("time_grid is rejected if in an invalid format", {
+    x <- data.frame(
+        vpt = c("b", "a", "a", "b", "c", "a"),
+        vtime = c(10, 20, 15, 5, 25, 35),
+        voutcome = c(2, 1, 4, 5, 6, 2)
+    )
+
+    # First check that it doesn't error when used properly
+    result <- tryCatch(
```

Comment:
Why is this needed? Would `expect_silent` be simpler?

Reply:
I guess normally we want the error from the test then, not just whether it errored. and since you fix it now logs are clean.